lib/bundles/inspec-compliance/api.rb

```diff
@@ -85,8 +88,12 @@ def self.version(config)
 
     # verifies that a profile
     def self.exist?(config, profile)
-      _msg, profiles = Compliance::API.profiles(config)
       owner, id, ver = profile_split(profile)
+
+      c = config.dup
```

Comment:
The `c` variable looks unused. Why is it needed? Looks like we can just delete lines 93 and 94, yes?